File: ios/Sources/CellContainerManager.m
#import <Foundation/Foundation.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(CellContainerManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(index, NSInteger)
RCT_EXPORT_VIEW_PROPERTY(stableId, NSString)

@end

File: ios/Sources/CellContainerManager.swift
import Foundation

@objc(CellContainerManager)
class CellContainerManager: RCTViewManager {  
    override func view() -> UIView! {
        return CellContainerComponentView()
    }
    
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
}
